collector/cfappscollector.go

```diff
@@ -17,16 +17,47 @@
 package collector
 
 import (
+	"fmt"
+	"net/url"
 	"os"
 	"path/filepath"
 	"strings"
 
+	cfclient "github.com/cloudfoundry-community/go-cfclient/v2"
 	"github.com/konveyor/move2kube/common"
+	"github.com/konveyor/move2kube/types"
 	collecttypes "github.com/konveyor/move2kube/types/collection"
 
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	inlineDepthRelations = "inline-relations-depth"
+	depth                = "2"
+)
+
+// CfCollectAppsMetadataKind defines kind of cf collect apps file
+const CfCollectAppsMetadataKind types.Kind = "CfCollectApps"
+
+// CfCollectApps defines definition of cf collect apps file
+type CfCollectApps struct {
+	types.TypeMeta `yaml:",inline"`
+	Spec           CfCollectAppsSpec `yaml:"spec,omitempty"`
+}
+
+// CfCollectAppsSpec stores the app information
+type CfCollectAppsSpec struct {
+	CfCollectApps []CfCollectApp `yaml:"applications"`
+}
+
+// CfCollectApp defines CfCollectApp information
+type CfCollectApp struct {
+	Name             string `yaml:"name"`
```

Comment:
Can't we make name also optional. Say somebody wants to specify just the guids.

Reply:
Yes, I have made name optional and the user can specify just the guids.